<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Ramakrishnan Periyakaruppan</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>openPOWERLINK MN Demo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/images.qrc">
    <normaloff>:/new/images/EpsgIcon.png</normaloff>:/new/images/EpsgIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_CDC"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPowerlink">
    <property name="title">
     <string>&amp;POWERLINK</string>
    </property>
    <addaction name="actionSelect_Interface"/>
    <addaction name="separator"/>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPowerlink"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>openPOWERLINK</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_CDC"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_Interface"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen_CDC">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/new/images/cdc.png</normaloff>:/new/images/cdc.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open CDC...</string>
   </property>
   <property name="toolTip">
    <string>Imports the CDC and XAP</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/new/images/exit.png</normaloff>:/new/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit the application</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionSelect_Interface">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/new/images/networkInterface.png</normaloff>:/new/images/networkInterface.png</iconset>
   </property>
   <property name="text">
    <string>Select &amp;Interface...</string>
   </property>
   <property name="toolTip">
    <string>Selects the network interface</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/new/images/start.png</normaloff>:/new/images/start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
   <property name="toolTip">
    <string>Starts the POWERLINK stack</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/new/images/stop.png</normaloff>:/new/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>S&amp;top</string>
   </property>
   <property name="toolTip">
    <string>Stops the POWERLINK stack</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources/images.qrc">
     <normaloff>:/new/images/KalycitoIcon.png</normaloff>:/new/images/KalycitoIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="iconText">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Help documentation</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/images.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
